src/components/molecules/NavBarScheduleWeather/NavBarScheduleWeather.tsx

```diff
@@ -41,15 +41,15 @@ export const NavBarScheduleWeather: React.FC<ContainerClassName> = ({
           )}
         >
           <div className="NavBarScheduleWeather__temp-and-state">
-            <h3>{`${weather.temperatureCelsius} °C`}</h3>
+            <h3>{`${weather.temperatureFahrenheit} °F`}</h3>
             <h5 className="NavBarScheduleWeather__state">
               {weather.stateOfSky}
             </h5>
           </div>
           <div>
             Humidity <b>{weather.humidityPercents}</b>% — Wind{" "}
             <b>
-              {weather.windSpeedKPH} KPH {weather.windDirection}
```

Comment:
do not remove, may be useful in the future,
put `const USE_IMPERIAL = true;`  just under the imports
with a @debt comment to extract it into util later on
use that flag to switch these two.

Reply:
https://github.com/sparkletown/sparkle/pull/2173#issuecomment-908932019